### DISCUSSION TITLE
'Additional' Fonts XXXL and XXXXL - are they working?!!

### DISCUSSION THREAD

**Discussion by DH106Comet** [FrSkyRC/ETHOS-Feedback-Community#4266] [Q&A]
I've just downloaded Ethos 1.5.11 onto an X20Pro and I'm having a lot of trouble accessing/using these two 'additional' fonts XXXL and XXXXL. I need to use them as I'm porting an older lua script from the Horus which has a lower screen resolution.

What's the current status regarding them? I've read other threads which state that they must be copied from the FLASH drive to the RADIO drive, and that they should be loaded in the 'build' event using lcd.loadFont( ).

I had some partial success in Ethos 1.5.8 - but only ONE of the two fonts would ever be useable (the first one to be loaded), the other would not work. Now that I've upgraded to 1.5.11, nothing I do seems to make them work :-(

**Comment by bsongis-frsky** [comment]
Issue created
